### DISCUSSION TITLE
dask.DataFrame support GeoDataFrame with GeoJSONSeq inputs

### DISCUSSION THREAD

**Discussion by dazza-codes** [dask/dask#8721] [Q&A] [answered]
Q. Can dask.DataFrame support GeoDataFrame datasets or GeoJSON Sequence inputs?

Given a very large set of output files in GeoJSON Sequence format [1], a geopandas.GeoDataFrame [2] can read each file OK.  It would be great to have some form of dask.DataFrame or dask.GeoDataFrame support to manage parallel workflows or streams on GeoJSON Sequence datasets.

[1] https://gdal.org/drivers/vector/geojsonseq.html 
[2] https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.html

**Comment by ian-r-rose** [comment, marked as answer]
Hi @dazza-codes, you'd probably be interested in looking at [this](https://github.com/geopandas/dask-geopandas) project, which is nearing its initial release. Also, this forum is no longer the recommended place to ask general usage questions for Dask, I'd recommend continuing the discussion over at http://dask.discourse.group

**Comment by dazza-codes** [comment]
What project?
- OH, `this` above is a link to https://github.com/geopandas/dask-geopandas (could not see it was a hyperlink in dark mode)

OK, I'll try to translate this to dask.discourse but this github feature is easy to convert to an issue, if this is a reasonable feature request?

Could dask.dataframe support the metadata for geo-data in parquet?

https://github.com/geopandas/geo-arrow-spec/blob/main/metadata.md

It does not decode WKB from a `geometry` field in a parquet output from geopandas.
